### PR TITLE
WIP 🌱 Adding bmh from the cluster template

### PR DESCRIPTION
**Pull request by mboukhalfa**
wip

### CODE REVIEW

**Comment by furkatgofurov7** [comment]
Will be removed, this is to test CI behaviour. 

/hold

File: test/e2e/data/infrastructure-metal3/cluster-template-ubuntu/kustomization.yaml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 resources:
+- ../bases/bmhs/node-0-bmc-secret.yaml
+- ../bases/bmhs/node-0.yaml
+- ../bases/bmhs/node-1-bmc-secret.yaml
+- ../bases/bmhs/node-1.yaml
```
Comment (mboukhalfa) [inline test/e2e/data/infrastructure-metal3/cluster-template-ubuntu/kustomization.yaml:5]: we need a way to call dynamic resources the number of bmh nodes is not fixed. The best way would be to load what ever files available in bmhs folder

**Comment by Rozzii** [comment]
/hold
This is titled as WIP but at some point lost the "hold" tag I am putting that back.

**Pull request closed without merging**